Create Table Scripts.txt

```diff
@@ -1,78 +1,78 @@
 );
 
 CREATE TABLE Appointment_Treatments (
-    AppointmentId INT,
-    TreatmentType VARCHAR(100) NOT NULL,
-    PRIMARY KEY (AppointmentId, TreatmentType),
-    FOREIGN KEY (AppointmentId) REFERENCES Appointment(AppointmentId) ON DELETE CASCADE
+    Appointment_Id INT,
+    Treatment_Type VARCHAR(100),
+    PRIMARY KEY (Appointment_Id, Treatment_Type),
+    FOREIGN KEY (Appointment_Id) REFERENCES Appointment(Appointment_Id) ON DELETE CASCADE
 );
 
-CREATE TABLE InsurancePolicy (
-    PolicyNumber VARCHAR(50) PRIMARY KEY,
-    CompanyName VARCHAR(255) NOT NULL,
-    CoverageLimit DECIMAL(10,2) NOT NULL,
-    StartDate DATE NOT NULL,
-    EndDate DATE NOT NULL,
-    PatientId INT,
-    FOREIGN KEY (PatientId) REFERENCES Patient(PatientId) ON DELETE CASCADE
+CREATE TABLE Insurance_Policy (
+    Policy_Number VARCHAR(50) PRIMARY KEY,
+    Company_Name VARCHAR(255) NOT NULL,
+    Coverage_Limit DECIMAL(10,2) NOT NULL,
+    Start_Date DATE NOT NULL,
+    End_Date DATE NOT NULL,
+    Patient_Id INT,
+    FOREIGN KEY (Patient_Id) REFERENCES Patient(Patient_Id) ON DELETE CASCADE
 );
 
 CREATE TABLE Referrer_Refers_Referral (
-    ReferralPatientId INT NOT NULL,
-    ReferrerPatientId INT NOT NULL,
-    PRIMARY KEY (ReferralPatientId, ReferrerPatientId),
-    FOREIGN KEY (ReferralPatientId) REFERENCES Patient(PatientId) ON DELETE NO ACTION,
-    FOREIGN KEY (ReferrerPatientId) REFERENCES Patient(PatientId) ON DELETE NO ACTION
+    Referral_Patient_Id INT,
+    Referrer_Patient_Id INT,
+    PRIMARY KEY (Referral_Patient_Id, Referrer_Patient_Id),
+    FOREIGN KEY (Referral_Patient_Id) REFERENCES Patient(Patient_Id) ON DELETE CASCADE,
+    FOREIGN KEY (Referrer_Patient_Id) REFERENCES Patient(Patient_Id) ON DELETE CASCADE
 );
 
 CREATE TABLE Payment (
-    PaymentId INT PRIMARY KEY,
-    PatientId INT,
-    AppointmentId INT,
-    FOREIGN KEY (PatientId) REFERENCES Patient(PatientId) ON DELETE CASCADE,
-    FOREIGN KEY (AppointmentId) REFERENCES Appointment(AppointmentId) ON DELETE NO ACTION
+    Payment_Id INT PRIMARY KEY,
+    Patient_Id INT,
+    Appointment_Id INT,
+    FOREIGN KEY (Patient_Id) REFERENCES Patient(Patient_Id) ON DELETE CASCADE,
+    FOREIGN KEY (Appointment_Id) REFERENCES Appointment(Appointment_Id) ON DELETE SET NULL
 );
 
 CREATE TABLE Payment_Details (
-    PaymentDetailID INT PRIMARY KEY,
+    Payment_Detail_Id INT PRIMARY KEY,
     Amount DECIMAL(8,2) NOT NULL,
-    PaymentDate DATE NOT NULL,
-    DueDate DATE NOT NULL,
-    PaymentType VARCHAR(50) NOT NULL,
-    PaymentStatus VARCHAR(50) NOT NULL,
-    InsuranceID VARCHAR(50),
-    PaymentID INT,
-    FOREIGN KEY (InsuranceID) REFERENCES InsurancePolicy(PolicyNumber) ON DELETE SET NULL,
-    FOREIGN KEY (PaymentID) REFERENCES Payment(PaymentId) ON DELETE NO ACTION
-);
\ No newline at end of file
+    Payment_Date DATE,
+    Due_Date DATE,
+    Payment_Type VARCHAR(50) NOT NULL,
+    Payment_Status VARCHAR(50) NOT NULL,
+    Policy_Number VARCHAR(50),

```

Comment:
I think it's easier to understand if the column name is the same as Insurance_Policy table.